packages/dd-trace/test/appsec/iast/taint-tracking/rewriter.spec.js

```diff
@@ -39,4 +39,46 @@ describe('IAST Rewriter', () => {
       expect(shimmer.unwrap.getCall(0).args[1]).eq('_compile')
     })
   })
+
+  describe('getOriginalPathAndLineFromSourceMap', () => {
+    let rewriter, getOriginalPathAndLineFromSourceMap, argvs
+    beforeEach(() => {
+      delete require.cache[require.resolve('../../../../src/appsec/iast/taint-tracking/rewriter')]
```

Comment:
why?

Reply:
to restore the initial state of the rewriter module. It caches the rewriter instance so it is only initialized once.

Reply:
but we are using `proxyquire` to require it, does it affect to `require.cache`?

Reply:
no it doesn't. Removed